iceoryx_examples/request_response_basic/CMakeLists.txt

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Build request_response_basic example
+cmake_minimum_required(VERSION 3.16)
+project(example_request_response_basic)
```

Comment:
Can we rename it to `example_request_response` or would you like to add multiple request response examples?
I would only differentiate between the languages, one example for c++ and one for c.